actions/check_cla.py

```
@@ -0,0 +1,26 @@
+# Ultralytics 🚀 AGPL-3.0 License - https://ultralytics.com/license
+
+import os
+
+
+def main():
+    """Check CLA on pull requests."""
+    print("RUNNING CHECK_CLA FUNCTION")
+    os.environ.get("GITHUB_TOKEN")
+    os.environ.get("CLA_REPOSITORY")
+    os.environ.get("CLA_BRANCH")
+    os.environ.get("CLA_SIGNATURES_PATH")
+    os.environ.get("CLA_DOCUMENT_URL")
+    os.environ.get("ALLOWLIST", "").split(",")
+    os.environ.get("SIGN_COMMENT")
+    os.environ.get("ALLSIGNED_COMMENT")
+
+    # Your CLA check logic here
+    # Note: You can extract the organization and repository name if needed:
+    # org, repo = cla_repository.split('/', 1) if '/' in cla_repository else ('', cla_repository)
+
+    # ... rest of your CLA check logic ...
+
```

❗ **CRITICAL**: `main()` exits successfully without consuming any of the inputs or performing a CLA check, so the action will never catch unsigned contributors. Please implement the verification logic (and fail fast when checks fail) before wiring this into workflows; otherwise every run will report success regardless of CLA status.